### PR TITLE
Diagrams

### PR DESCRIPTION
**Pull request by thomas-fossati**

### CODE REVIEW

File: tls-attestation/collaterals/integration-passport.puml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+participant "TLS B" as tls_b order 30
+participant "PARSEC" as parsec order 50
+participant "KAS" as kas order 50
+
+group KAT discovery
+tls_b --> parsec: get_kat_media_type()
+tls_b <-- parsec: kat-media-type
+end
+
+group Passport issuance
```
Comment (ionut-arm) [inline tls-attestation/collaterals/integration-passport.puml:15]: Sticking with the "TLS B participates in the issuance, and stores the passport" approach? (Not criticising, just curious if TLS B essentially means client app + TLS stack essentially)

File: tls-attestation/diagrams/background-check-tls-handshake.txt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+|  |   nonce,            |                        |               |    |
+|  |   supp-media-types  |                        |               |    |
+|  | }                   |                        |               |    |
+|  +-------------------->|                        |               |    |
+|  |                     | ClientHello            |               |    |
+|  |                     |  {...}                 |               |    |
+|  |                     |  attested-key-ext-req( |               |    |
+|  |                     |    nonce,              |               |    |
+|  |                     |    supp-media-types    |               |    |
+|  |                     |  )                     |               |    |
```
Comment (ionut-arm) [inline tls-attestation/diagrams/background-check-tls-handshake.txt:22]: Are the arrows between TLS A and TLS B missing here?

File: tls-attestation/diagrams/kat-media-type-discovery.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.-------.      .------------.          .-------.       .----------.
+| TLS A |      | Verifier X |          | TLS B |       | Attester |
+'---+---'      '-----+------'          '---+---'       '----+-----'
```
Comment (ionut-arm) [inline tls-attestation/diagrams/kat-media-type-discovery.txt:3]: Hmm, do we need TLS A and Verifier? I assume it's just for completeness

**Review by ionut-arm — COMMENTED** [review]
Looks great! a handful of comments

File: tls-attestation/diagrams/background-check-tls-handshake.txt (hunk 2 above)
Comment (thomas-fossati) [reply tls-attestation/diagrams/background-check-tls-handshake.txt:22]: Eagle eye! Fixed in [33d969b](https://github.com/hannestschofenig/tschofenig-ids/pull/53/commits/33d969b70480ce8966b6957d554def3f12998f9e)

File: tls-attestation/diagrams/kat-media-type-discovery.txt (hunk 3 above)
Comment (thomas-fossati) [reply tls-attestation/diagrams/kat-media-type-discovery.txt:3]: I imagined we'd use them in sequence, with text blocks in between.  This is a way to provide the continuity across boundaries.

**Pull request merged**